net.hypatiaproject.Hypatia.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id" : "net.hypatiaproject.Hypatia",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "hypatia",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak"
```

Comment:
why is this needed ?

Reply:
this needs an exception https://docs.flathub.org/docs/for-app-authors/linter#exceptions

error code is https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

Reply:
the old app didn't have this.